legend-pure-code-compiled-core/src/main/resources/core/pure/protocol/vX_X_X/transfers/valueSpecification.pure

```diff
@@ -93,7 +94,8 @@ function meta::protocols::pure::vX_X_X::transformation::fromPureGraph::valueSpec
                                                        |  let xs = $fe.parametersValues
                                                                                        ->filter(f|$f->instanceOf(InstanceValue))->cast(@InstanceValue).values
                                                                                         ->filter(f|$f->instanceOf(LambdaFunction))->cast(@LambdaFunction<Any>).expressionSequence;
-                                                           let newProperty = $xs->at(0) ->cast(@SimpleFunctionExpression);
+                                                           let newProp = $xs->at(0) ->cast(@SimpleFunctionExpression);
```

Comment:
No. They actually resolve automap a stage before they process property so that they have the info about the previous processed property to replace the automap with.